src/components/electricity_tariffs/energycharts/electricity_tariff.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+	<div class="electricity-tariff-energycharts">
+		<openwb-base-select-input
+			title="Land"
+			notSelected="Bitte auswählen"
+			:options="[
+				{ value: 'de', text: 'Deutschland' },
+				{ value: 'at', text: 'Österreich' },
```

Comment:
```suggestion
				{ value: 'DE-LU', text: 'Deutschland' },
				{ value: 'AT', text: 'Österreich' },
```
So kannst Du dir das matching im Backend sparen.

Wir brauchen als Länder nur Deutschland, Österreich und Schweiz. In anderen Ländern wird die openWB nicht vertreiben.